### DISCUSSION TITLE
Styling with Carbon Design

### DISCUSSION THREAD

**Discussion by Cslamonde** [carbon-design-system/carbon-components-svelte#1681] [Q&A] [answered]
I'm building an e-commerce web app and I'm having a challenge structuring fetched data into columns and rows using carbon design. How do I add a picture, information like price, category and description together with a buttons that are responsive in every device.
@metonym

**Comment by metonym** [comment]
This is a fairly broad question but I'm happy to link some resources for you to get started.

For displaying data in "columns and rows," I would recommend starting with the [`DataTable` component ](https://carbon-components-svelte.onrender.com/components/DataTable)to see if it fits your needs. It accepts data in a tabular format and supports the basics functions you would expect from a table, like sorting, filtering, expansion, pagination etc..

The [Carbon Tutorial for Svelte](https://github.com/carbon-design-system/carbon-tutorial-svelte) has a an [example component](https://github.com/carbon-design-system/carbon-tutorial-svelte/blob/main/src/routes/repos.svelte) that you can reference. It fetches data in the `onMount` hook and then renders a paginated table with the response.

**Reply by Cslamonde** [reply]
Lemme narrow it down. If you visit the [Alibaba website](https://www.alibaba.com/), you'll see how the data has been displayed. If you scroll through the products, you can see a lit bit of the information like a picture, a brief description and the price. In addition, it is responsive. How can I achieve that?

**Comment by Cslamonde** [comment]
@metonym, Is there a card component in v.11?

**Reply by metonym** [reply, marked as answer]
The closest would be the [Tile component](https://carbon-components-svelte.onrender.com/components/Tile).

**Reply by Cslamonde** [reply]
Thanks
